roles/swift-ring/handlers/main.yml

```diff
@@ -1,5 +1,19 @@
 ---
-- name: setup swift rings
+- name: setup swift rings - bootstrap
   command: sudo -u swiftops /usr/local/bin/swifttool -i /home/swiftops/.ssh/id_rsa
-           bootstrap --config /etc/swift/ring_definition.yml --outdir /etc/swift
+           --config {{ swift_ring.outdir }}/ring_definition.yml --outdir {{ swift_ring.outdir }} bootstrap
+
+- name: setup swift rings - scaleup
+  command: sudo -u swiftops /usr/local/bin/swifttool -i /home/swiftops/.ssh/id_rsa
+           --config {{ swift_ring.outdir }}/ring_definition.yml --outdir {{ swift_ring.outdir }} scaleup
+           --iterations {{ swift_ring.iterations }} 
+  async: "{{ swift_ring.iterations * 60 * 60|int }}"
+  poll: 0
```

Comment:
I see, okay. So given that the babysitter will kill the process group at the timeout value, is this time long enough? What happens if swift tool is killed before it is done?